warehouse/internal/repo/staging.go

```diff
@@ -49,11 +51,10 @@ type metadataSchema struct {
 }
 
 func StagingFileIDs(stagingFiles []*model.StagingFile) []int64 {
-	var stagingFileIDs []int64
-	for _, stagingFile := range stagingFiles {
-		stagingFileIDs = append(stagingFileIDs, stagingFile.ID)
-	}
-	return stagingFileIDs
+	return lo.Map(
+		stagingFiles,
+		func(stagingFile *model.StagingFile, _ int) int64 { return stagingFile.ID },
+	)
```

Comment:
Subjective comment, but not in favour of using `lo` here. I find the previous code simple enough.

We could do, for a minor performance improvement:
```suggestion
        stagingFileIDs := make([]int64, len(stagingFiles)
	for i, stagingFile := range stagingFiles {
		stagingFileIDs[i] = stagingFile.ID
	}
```